docs/src/modules/components/AppLayoutDocs.js

```diff
@@ -120,23 +120,27 @@ export default function AppLayoutDocs(props) {
 
   const { canonicalAs } = pathnameToLanguage(router.asPath);
   let productName = 'MUI';
+  let teamName = 'Core';
   if (canonicalAs.startsWith('/material-ui/')) {
     productName = 'Material UI';
   } else if (canonicalAs.startsWith('/base-ui/')) {
     productName = 'Base UI';
   } else if (canonicalAs.startsWith('/x/')) {
     productName = 'MUI X';
+    teamName = 'X';
   } else if (canonicalAs.startsWith('/system/')) {
     productName = 'MUI System';
   } else if (canonicalAs.startsWith('/toolpad/')) {
     productName = 'MUI Toolpad';
+    teamName = 'Toolpad';
   } else if (canonicalAs.startsWith('/joy-ui/')) {
     productName = 'Joy UI';
   }
 
   const Layout = disableLayout ? React.Fragment : AppFrame;
   const layoutProps = disableLayout ? {} : { BannerComponent };
 
+  const card = `https://deploy-preview-41188--material-ui-apps.netlify.app/og-docs?product=${teamName}&folder=${productName}&page=${title}`;
```

Comment:
Equivalent?

```suggestion
  const card = `/og-docs?product=${teamName}&folder=${productName}&page=${title}`;
```

Reply:
Ah ok, it's because of https://stackoverflow.com/questions/9858577/open-graph-can-resolve-relative-url. I forgot I added this.

Reply:
Then maybe https://github.com/mui/material-ui/blob/f58d02eaa5929f48213debe124c130f367e864a4/docs/src/modules/components/DemoToolbar.js#L406

or I guess even better `DEPLOY_PRIME_URL` https://docs.netlify.com/configure-builds/environment-variables/

Reply:
I solved it with this in the `Header.tsx` such that I can use relative path that will be correctly resolved :+1:
```
// #major-version-switch
const HOST = process.env.PULL_REQUEST_ID
  ? `https://deploy-preview-${process.env.PULL_REQUEST_ID}--${process.env.NETLIFY_SITE_NAME}.netlify.app`
  : 'https://mui.com';
```